### DISCUSSION TITLE
Documentation for non-RGB accessors?

### DISCUSSION THREAD

**Discussion by dheera** [blaylockbk/goes2go#67] [Q&A]
I'm having trouble accessing data for `ABI-L1b`. It says the `rgb` accessor isn't available, and there isn't an "ir" accessor. Is there documentation for how to access, say, Band 6 and plot it?

**Comment by blaylockbk** [comment]
Hi @dheera 

By design, only the `ABI-L2-MCMIP` (multi-channel cloud and moisture imagery) product works with the RGB accessor. The reason is because this product contains all 16 channels within the same file on the same fixed ABI grid, and different RGB recipes require different channels. I admit, this isn't super clear in the docs aside from the examples and could be improved.

If you want to get and plot just channel six you can try any of these:

### Level 1 Radiance data, GOES-16
```python
from goes2go import GOES
import matplotlib.pyplot as plt

G = GOES(satellite=16, product="ABI-L1b-Rad", domain="C", channel=6)
df = G.latest()
plt.imshow(df.Rad)
plt.title(f"{df.title}\nBand:{df.band_id.item()}", loc="left")
plt.title(df.t.dt.strftime("%H:%M UTC %d %b %Y").item(), loc="right")
```
![image](https://github.com/blaylockbk/goes2go/assets/6249613/e466be95-df6d-4a9c-b583-080d2936e930)

### Level 2 Cloud Moisture Imagry Product; GOES-18
```python
G = GOES(satellite=18, product="ABI-L2-CMIP", domain="C", channel=6)
df = G.latest()
plt.imshow(df.CMI)
plt.title(f"{df.title}\nBand:{df.band_id.item()}", loc="left")
plt.title(df.t.dt.strftime("%H:%M UTC %d %b %Y").item(), loc="right")
```
![image](https://github.com/blaylockbk/goes2go/assets/6249613/97b78e01-f4b7-478a-85cf-77d78a933813)

### Level 2 _Multi-channel_ Cloud Moisture Imagery; GOES-16
Remember, this file has all 16 channels, so the variables name is slightly different.
```python
G = GOES(satellite=16, product="ABI-L2-MCMIP", domain="C")
df = G.latest()
plt.imshow(df.CMI_C06)
plt.title(f"{df.title}\nBand:{df.band_id_C06.item()}", loc="left")
plt.title(df.t.dt.strftime("%H:%M UTC %d %b %Y").item(), loc="right")
```
![image](https://github.com/blaylockbk/goes2go/assets/6249613/10df3aa1-916b-4721-90a7-e4de3734bdb7)
